simpletest/test.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+from os import path, makedirs
+import p2ptrans as p2p
+from pylada.crystal import read
+from test_utils import *
+import pickle
+
+
+if __name__ == "__main__":
+    bcc = read.poscar(path.join('ref','BCC_Fe_POSCAR'))
+    fcc = read.poscar(path.join('ref','FCC_Fe_POSCAR'))
+
+    outdir = 'results'
+    if not path.exists(outdir):
+        makedirs(outdir) 
+
+    # n_steps, (tmat, dispStruc, vec_classes), outdir, display
+    tmat, dispStruc, vec_classes = p2p.findMatching(bcc, fcc, 100, outdir=outdir)
+    p2p.produceTransition(60, tmat, dispStruc, vec_classes, outdir, False)
```

Comment:
produceTransition returns transStruc, spgList, Tpos, color_array, atom_types transStruc is the list of pylada structures that are intermediate steps of the transformation, so you don't need to read the POSCARS for TransPOSCARS